app/component/map/MapWithTracking.js

```diff
@@ -295,6 +295,12 @@ class MapWithTrackingStateHandler extends React.Component {
       : this.state.mapTracking
         ? 'icon-tracking-on-v2'
         : 'icon-tracking-offline-v2';
+    // eslint-disable-next-line no-nested-ternary
+    const ariaLabel = position.locationingFailed
+      ? 'tracking failed'
+      : this.state.mapTracking
+        ? 'tracking on'
+        : 'tracking off';
```

Comment:
Reittiopas supports many languages. Here, you set always english labels. Add appropriate messages to app/translations.js and translate  the label as in other components. See e.g. SecondaryButton.js.